pep-0001.txt

```diff
@@ -242,37 +245,86 @@ on GitHub.
+
+* The forum is appropriate to the PEP's topic.
+* The thread is publicly available on the web so that all interested parties
+  can participate.
+* The discussion is subject to the `Python Community Code of Conduct
+  <https://www.python.org/psf/conduct/>`_.
+* A direct link to the current discussion thread is provided in the PEP
+  under the ``Discussions-To`` header.
+
+Typically, the `Python-Dev`_ mailing list and the
+`PEPs category`_ of the `Python Discourse`_ are good choices for most PEPs,
+while some specialized topics have specific venues, such as
+`Typing-SIG`_ for typing PEPs or the `Packaging category`_ on the Python
+Discourse for packaging PEPs. If the PEP authors are unsure of the best venue,
+the PEP Sponsor and PEP editors can advise them accordingly.
+
+If the chosen venue is not the `Python-Dev`_ mailing list,
+a brief announcement should be posted there when the draft PEP is
+committed to the PEP repository and available on the PEP website,
+with a link to the rendered PEP and the to canonical Discussions-To thread.
+
+If a PEP undergoes a significant re-write or other major, substantive
+changes to its proposed specification, a new thread should typically be created
+in the chosen venue to solicit additional feedback. If this occurs, the
+``Discussions-To`` link and ``Post-History`` in the PEP must be updated to
+reflect this new thread, and (if not the discussion venue) a further
+announcement sent to `Python-Dev`_ containing the same information as above
+and at least briefly summarizing the major changes.
+
 PEP authors are responsible for collecting community feedback on a PEP
-before submitting it for review. However, wherever possible, long
-open-ended discussions on public mailing lists should be avoided.
-Strategies to keep the discussions efficient include: setting up a
-separate SIG mailing list for the topic, having the PEP author accept
-private comments in the early design phases, setting up a wiki page, etc.
+before submitting it for review. However, to avoid long-winded and
+open-ended discussions, strategies such as soliciting private or more
+narrowly-tailored feedback in the early design phase,
+collaborating with other community members with expertise in the PEP's
+subject matter, and picking an appropriately-specialized discussion for the
+PEP's topic (if applicable) should be considered.
 PEP authors should use their discretion here.
 
+Once the PEP is assigned a number and committed to the PEP repository,
+substantive issues should generally be discussed on the canonical public
+thread, as opposed to private channels, GitHub pull request reviews or
+unrelated venues. This ensures everyone can follow and contribute,
+avoids fragmenting the discussion,
+and makes sure it is fully considered as part of the PEP review process.
+Comments, support, concerns and other feedback on this designated thread
+are a critical part of what the Steering Council or PEP-Delegate will
+consider when reviewing the PEP.
```

Comment:
Earlier it was mentioned that the "designated thread" may need to be updated/changed over time. Thus links to the earlier threads will be lost in the latest version of the PEP document.

Is it likely the case that the SC will want to go back and read not just the lastest designated thread, but perhaps _all_ prior such threads? If so, then perhaps the SC would find it useful to keep **multiple** threads in the Discussions-To field?

Reply:
That's a great point (albeit not really a regression), and the plan to tackle it was by inline-linking each date in `Post-History` to that respective thread, making that field have a much clearer and more practically useful purpose following these changes. I almost included that change here as well, but since it was more purely technical in nature and a little out of the original scope, I decided to defer it to a followup PR once this one was in. I can open the followup PR with that change shortly, and include your other fixes as well.